src/algebra/group/inj_surj.lean

```diff
@@ -179,6 +179,17 @@ protected def comm_monoid [comm_monoid M₂] (f : M₁ → M₂) (hf : injective
   comm_monoid M₁ :=
 { .. hf.comm_semigroup f mul, .. hf.monoid f one mul npow }
 
+/-- A type endowed with `0`, `1` and `+` is an additive commutative monoid with one,
+if it admits an injective map that preserves `0`, `1` and `+` to an additive commutative monoid with
+one. -/
+@[reducible] -- See note [reducible non-instances]
```

Comment:
It looks like in the rest of this file this comment goes in the docstring; so probably we should be consistent.